src/pages/Embeds/Plio.vue

```diff
@@ -703,6 +705,19 @@ export default {
       // Disabling autoplay because of bug - issue #157
       // this.playPlayer();
     },
+    /**
+     * Places the given marker at a defined position on the plyr progress bar
+     * Also sets it's custom style classes
+     * @param {Object} progressBar - The HTML element representing the progress bar
+     * @param {Object} marker - The HTML element that needs to be placed on the progress bar
+     * @param {Array} classList - An array of tailwind classes
+     * @param {Number} positionPercent - By what % from the left should the marker be placed
+     */
+    placeMarkerOnSlider(progressBar, marker, classList, positionPercent) {
```

Comment:
move logic for replacing existing marker and checking if progress bar exists to this function as well